tasks/dashboards.yml

```diff
@@ -123,3 +137,53 @@
 #   no_log: true
 #   with_fileglob:
 #     - "/tmp/dashboards/*"
+
+- name: Create/Update dashboards file (provisioning)
+  become: true
+  copy:
+    dest: "/etc/grafana/provisioning/dashboards/ansible.yml"
+    content: |
+      apiVersion: 1
+      providers:
+       - name: 'default'
+         orgId: 1
+         folder: ''
+         type: file
+         options:
+           path: /var/lib/grafana/dashboards
+    backup: false
+    owner: root
+    group: grafana
+    mode: 0640
+  notify: restart grafana
+  when: grafana_use_provisioning
+
+- name: Import grafana dashboards through provisioning
+  synchronize:
+    src: "/tmp/dashboards/"
+    dest: "/var/lib/grafana/dashboards"
+    archive: false
+    checksum: true
+    recursive: true
+    delete: "{{ grafana_provisioning_synced }}"
+    rsync_opts:
+      - "--no-motd"
+  when: grafana_use_provisioning
+  register: synchronize_result
+
+- name: Set privileges on provisioned dashboards
```

Comment:
Since this task and next one are only executed when previous one was changed, then it might be better to use those as handlers instead of using `when: synchronize_result.changed` and registering variable. In the end that's what handlers are for. I don't know why this wasn't showing up in CI, but when I run it locally I immediately got linter errors:
```
    [ANSIBLE0016] Tasks that run when changed should likely be handlers
    /home/paulfantom/cloudalchemy/ansible-grafana/tasks/dashboards.yml:174
    Task/Handler: Set privileges on provisioned dashboards
    
    [ANSIBLE0016] Tasks that run when changed should likely be handlers
    /home/paulfantom/cloudalchemy/ansible-grafana/tasks/dashboards.yml:183
    Task/Handler: Set privileges on provisioned dashboards directory
```

Reply:
Oh yes I see. I never used handlers for this kind of situation. Can I attach multiple tasks to one handler name ?

Reply:
You can notify multiple handlers from one task like:

```
- name: doing sth
  command: echo "OK"
  notify:
    - do thing A
    - do thing B
```